EEBOS/SCROWL
# Chore/typings asset modules [sc-631]

### Short summary

- Moved all configs to config package
- Simplified linting setup of apps from config files to config objects in the root packages

---

### Test steps

1. `yarn start:tool:course` should start the app as normal
2. 'yarn deploy` should build the executable and the installed app should run as normal
